judgments/tests/tests.py

```diff
@@ -206,6 +209,21 @@ def test_judgment_results(self, fake_result, fake_advanced_search):
             response, '<meta name="robots" content="noindex,nofollow" />'
         )
 
+    @patch("judgments.views.results.perform_advanced_search")
+    @patch("judgments.models.SearchResult.create_from_node")
+    def test_static_page(self, fake_result, fake_advanced_search):
+        fake_advanced_search.return_value = fake_search_results()
+        fake_result.return_value = fake_search_result()
+        # The judgment search results page should have a robots meta tag
+        # with nofollow,noindex
+        response = self.client.get("/what-to-expect")
+        self.assertNotContains(
+            response, '<meta name="robots" content="noindex,nofollow" />'
```

Comment:
This might be better a bit fuzzier, just checking it doesn't contain `noindex`, so that if we make eg whitespace changes it doesn't break the test.

Reply:
good plan!